.travis.yml

```diff
@@ -1,20 +1,44 @@
 language: ruby
 rvm:
   - 2.3.1
+
 branches:
   only:
     - master
     - develop
-script: bundle exec rspec --tag ~broken_in_ci
+
+services:
+  - mysql
+  - postgresql
```

Comment:
If there is, it's not documented.  I tried a couple of the obvious things and couldn't get it to work.